docs/backup-modules.md

```diff
@@ -1,77 +1,64 @@
-# Backing up of existing modules
+# Backing up of existing modules (`--backup-modules`)
 
-!!! warning
-    This page will soon replace <https://docs.easybuild.io/en/latest/Backup_modules.html>.
+While regenerating existing module files, you may want to preserve the
+existing module files to compare and assess that the changes in the
+newly generated module file match expectations.
 
-    **
-    It still needs to be ported from *reStructuredText* (.rst) to *MarkDown* (.md),  
-    and you can help with that!
-    **
+Backing up of existing modules can be enabled with `--backup-modules`.
 
-    - source: [`docs/Backup_modules.rst` in `easybuilders/easybuild` repo](https://raw.githubusercontent.com/easybuilders/easybuild/develop/docs/Backup_modules.rst)
-    - target: [`docs/backup-modules.md` in `easybuilders/easybuild-docs` repo](https://github.com/easybuilders/easybuild-docs/tree/main/docs/backup-modules.md)
+Backups are stored in the same directory as where the module file was
+located, and the files are given an additional extension of the form
+`.bak_<year><month><day><hour><min><sec>`.
 
-    See <https://github.com/easybuilders/easybuild-docs> for more information.
+- With module files in Tcl syntax, the backup module file is hidden by
+adding a leading dot to the filename; this is done to avoid that it
+is displayed as a normal module in `module avail`.
```

Comment:
To fix the failing MarkDown linter check, you should indent here:

```
- With module files in Tcl syntax, the backup module file is hidden by
  adding a leading dot to the filename; this is done to avoid that it
  is displayed as a normal module in `module avail`.
```

Reply:
@boegel interestingly my linter didn't mind this, although I was surprised! Is your preference here to split the lines or should I just do each point as a single line and are you enforcing any line-length limits?